instrumentation/elasticsearch/test/instrumentation/elasticsearch/patches/client_test.rb

```diff
@@ -0,0 +1,86 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+require 'test_helper'
+
+require_relative '../../../../lib/opentelemetry/instrumentation/elasticsearch'
+require_relative '../../../../lib/opentelemetry/instrumentation/elasticsearch/patches/client'
+
+describe OpenTelemetry::Instrumentation::Elasticsearch::Patches::Client do
+  let(:instrumentation) { OpenTelemetry::Instrumentation::Elasticsearch::Instrumentation.instance }
+  let(:exporter) { EXPORTER }
+  let(:span) { exporter.finished_spans.first }
+
+  let(:host) { ENV.fetch('TEST_ELASTICSEARCH_HOST', '127.0.0.1') }
+  let(:port) { ENV.fetch('TEST_ELASTICSEARCH_PORT', '9200').to_i }
+  let(:server) { "http://#{host}:#{port}" }
+  let(:client) { Elasticsearch::Client.new(url: server) }
+  let(:url) { "http://#{host}:#{port}/#{path}" }
+
+  before do
+    exporter.reset
+    instrumentation.install
+  end
+
+  after do
+    instrumentation.instance_variable_set(:@installed, false)
+  end
+
+  describe 'search query' do
+    let(:path) { '_search?q=test' }
+    it 'produces correct traces' do
+      stub_request(:get, url).to_return(status: 200)
+
+      client.search q: 'test'
+
+      _(span.name).must_equal 'elasticsearch.query'
+      _(span.attributes['out.host']).must_equal host
+      _(span.attributes['out.port']).must_equal port
```

Comment:
```suggestion
      _(span.attributes['net.peer.port']).must_equal port
```
https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/span-general.md#general-network-connection-attributes